### ISSUE TITLE
Set Dashboard filter box value before email report?

### ISSUE THREAD

**Issue by waytoyou** [apache/superset#15367]
In the dashboard  I put filter box to control the data in table. When I sent report via email. The report shows the report by default value in filter box. Is there any solution to email  different report? Thanks a lot.

**Comment by amitmiran137** [comment]
We should leverage filtersets for choosing filters in a reports for scheduling purposes

With that dashboard will load with all the nessecery filters and then screenshot will be taken

**Comment by waytoyou** [comment]
@amitmiran137 Yeah!!! Now report settings seems can not leverage any filtersets.

**Comment by VincentZ112** [comment]
Follow up on this ticket. I am trying to send different customers the same report with different filters. This feature could be really helpful.

**Comment by DonWizu** [comment]
I subscribe to the thread as well. Quite interested on this feature. Having the same issues as we need to send reports with different filters. So far the only solution is to create two dashboards with different default filters and create two different reports, but it's harder to maintain.
